jollystation_modules/code/modules/mob/living/carbon/human/species_types/skrell.dm

```diff
@@ -68,7 +68,7 @@ GLOBAL_LIST_EMPTY(head_tentacles_list)
 	if(istype(human.wear_mask) && (human.wear_mask.flags_inv & HIDEHAIR))
 		. = FALSE
 	var/obj/item/bodypart/head/our_head = human.get_bodypart(BODY_ZONE_HEAD)
-	if(our_head?.status == BODYTYPE_ROBOTIC)
+	if(our_head?.bodytype == BODYTYPE_ROBOTIC)
```

Comment:
this is a bitflag not a single value.

`if(ourhead && !IS_ORGANIC_LIMB(ourhead)`